Translations update from Weblate

weblate opened the pull request:
Translations update from [Weblate](https://hosted.weblate.org/projects/todont/to-dont/) for To Don't/To Don't.



Current translation status:

![Weblate translation status](https://hosted.weblate.org/widgets/todont/-/to-dont/horizontal-auto.svg)

Commit:
Translated using Weblate (Indonesian)

Currently translated at 100.0% (7 of 7 strings)

Translation: To Don't/Metadata
Translate-URL: https://hosted.weblate.org/projects/todont/metadata/id/

CrazyMarvin commented:
If a file in the ```fastlane``` folder, like the title.txt or video.txt for example, stays the same, it shouldn't be added.

Commit:
Translated using Weblate (Esperanto)

Currently translated at 100.0% (2 of 2 strings)

Translation: To Don't/To Don't
Translate-URL: https://hosted.weblate.org/projects/todont/to-dont/eo/

Commit:
Translated using Weblate (Chinese (Simplified))

Currently translated at 100.0% (2 of 2 strings)

Translation: To Don't/To Don't
Translate-URL: https://hosted.weblate.org/projects/todont/to-dont/zh_Hans/

Commit:
Translated using Weblate (Norwegian Bokmål)

Currently translated at 100.0% (2 of 2 strings)

Translation: To Don't/To Don't
Translate-URL: https://hosted.weblate.org/projects/todont/to-dont/nb_NO/

Commit:
Translated using Weblate (Polish)

Currently translated at 100.0% (2 of 2 strings)

Translation: To Don't/To Don't
Translate-URL: https://hosted.weblate.org/projects/todont/to-dont/pl/

Commit:
Added translation using Weblate (Bulgarian)

Commit:
Translated using Weblate (Bulgarian)

Currently translated at 100.0% (2 of 2 strings)

Translation: To Don't/To Don't
Translate-URL: https://hosted.weblate.org/projects/todont/to-dont/bg/

Commit:
Added translation using Weblate (Danish)

Commit:
Translated using Weblate (Danish)

Currently translated at 100.0% (2 of 2 strings)

Translation: To Don't/To Don't
Translate-URL: https://hosted.weblate.org/projects/todont/to-dont/da/

Commit:
Added translation using Weblate (Estonian)

Commit:
Translated using Weblate (Estonian)

Currently translated at 100.0% (2 of 2 strings)

Translation: To Don't/To Don't
Translate-URL: https://hosted.weblate.org/projects/todont/to-dont/et/

Commit:
Added translation using Weblate (Finnish)

Commit:
Translated using Weblate (Finnish)

Currently translated at 100.0% (2 of 2 strings)

Translation: To Don't/To Don't
Translate-URL: https://hosted.weblate.org/projects/todont/to-dont/fi/

Commit:
Translated using Weblate (Romanian)

Currently translated at 100.0% (2 of 2 strings)

Translation: To Don't/To Don't
Translate-URL: https://hosted.weblate.org/projects/todont/to-dont/ro/

Commit:
Added translation using Weblate (Slovenian)

Commit:
Added translation using Weblate (Slovak)

CrazyMarvin committed:
Delete title.txt
